clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1176,7 +1176,8 @@ public void assign(Collection<TopicPartition> partitions) {
      * offset for the subscribed list of partitions
      *
      * <p>
-     * This method returns immediately if there are records available or if the position advances past control records.
+     * This method returns immediately if there are records available or if the position advances past control records
+     * or aborted transactions when isolation.level=READ_COMMITTED.
```

Comment:
nit: Worth double-checking, but I think we require "read_committed" to be lower case.

Reply:
Sure, https://github.com/apache/kafka/blob/a2da309f134ec2930f70f1a3917eaade7548132f/clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java#L557.

Reply:
```suggestion
     * or aborted transactions when isolation.level=read_committed.
```